modules/ssh-agent-using.adoc

```diff
@@ -93,6 +93,11 @@ If you want to perform installation debugging or disaster recovery on your clust
 In a production environment, you require disaster recovery and debugging.
 ====
 
+[NOTE]
```

Comment:
I don't love the placement of this note, because it puts two notes back-to-back, and there is a third really close. Can we see if there is a better location for this? Maybe to the first step which is actually generating the SSH key?

Reply:
Agree, pushed changes to move this note to the bottom of 2nd step. Looks much nicer.